flake.nix

```diff
@@ -5,14 +5,18 @@
     #   - https://discourse.nixos.org/t/differences-between-nix-channels/13998
     # How to update the revision
     #   - `nix flake update --commit-lock-file` # https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-flake-update.html
-    # TODO: Use nixpkgs-24.05-darwin only in macOS. See GH-910
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
-    edge-nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    edge-nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable"; # Unfit for darwin, might be broken. See https://github.com/NixOS/nixpkgs/issues/107466
+    nixpkgs-darwin.url = "github:NixOS/nixpkgs/nixpkgs-24.05-darwin";
```

Comment:
If you reach this repository with searching `08gjpy7gfpylh2m2dyhasd0l5i3y7mz2-ghostscript-10.04.0.drv`, this line is the answer by me 🤷‍♂️ 🍎